### PR TITLE
Use prettier to format package-lock.json with spaces

### PR DESCRIPTION
**Pull request by cjwooo**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "bracketSpacing": false,
-  "parser": "typescript",
```
Comment (cjwooo) [inline .prettierrc.json:3]: Prettier will automatically the parser based on filetype.

File: package.json [hunk 2]
```diff
@@ -45,5 +45,15 @@
     "ts-jest": "^27.1.3",
     "ts-node": "^10.6.0",
     "typescript": "^4.6.2"
+  },
```
Comment (cjwooo) [inline package.json:48]: With this patch, `npm i` will automatically install the git pre-commit hook that formats the package-lock.json if it is staged.

**Comment by tovbinm** [comment]
Awesome! Do we have this problem elsewhere? After this PR is merged, we can now approve and merge Dependabot PRs?

**Comment by cjwooo** [comment]
I hope so.

**Pull request merged**